packages/terra-heading/docs/heading-visually-hidden.md

```diff
@@ -0,0 +1,6 @@
+## Visually Hidden Headings
+
+The `isVisuallyHidden` prop can be used to hide the content visually, yet provide the content to screen readers.
+More info on this approach can be found here: http://a11yproject.com/posts/how-to-hide-content/
+
+The example below includes the following `<Heading />` component using the `isVisuallyHidden` prop.
```

Comment:
Updated: https://github.com/cerner/terra-core/pull/470/commits/286b3bc3f2998714cd1aa971c4c7120e9689b622

I moved this text into the example.